solr/bin/solr.cmd

```diff
@@ -1525,10 +1524,10 @@ IF "!ZK_OP!"=="" (
 set CONNECTION_PARAMS=""
 
 IF "!ZK_OP!"=="" (
-  set CONNECTION_PARAMS="-solrUrl !ZK_SOLR_URL!"
+  set CONNECTION_PARAMS="--solr-url !ZK_SOLR_URL!"
```

Comment:
good catch!   I suspect because we still support `-solrUrl` that is why this works...

Reply:
For deprecation reasons I assume this should be updated to the new format, correct?